src/tests/frontend/specs/focus.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+describe('caret focus', function () {
+  describe('doesnt focus on editor using focusOnEditor URL Param', function () {
+    before(async function () {
+      await helper.aNewPad({
+        params: {
+          focusOnEditor: 'false'
+        }
+      });
+    });
+
+    it('does not focus when focusOnEditor is set to false', async function () {
+      const activeElement = helper.padChrome$.document.activeElement;
+      await helper.waitForPromise(() => $(activeElement).hasClass('readwrite'));
```

Comment:
Waiting for the `readwrite` class might not be long enough. See my comment for the `before()`.

Also, this doesn't actually test that it does not focus because the `activeElement` variable is set right away but might be updated later. Fix (when combined with my above suggested change):

```suggestion
      expect(helper.padChrome$.document.activeElement)
          .to.not.be(helper.padChrome$.document.querySelector('iframe[name="ace_outer"]'));
```